CMakeLists.txt

```diff
@@ -11,7 +11,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 
 PROJECT(tiny_cnn)
-set(PROJECT_VERSION "0.0.1")
+set(PROJECT_VERSION "0.1.0")
 string(REGEX MATCHALL "[0-9]" PROJECT_VERSION_PARTS "${PROJECT_VERSION}")
```

Comment:
@nyanp Well yes, you are right, there's no coordination between branches. We discussed it in other thread. https://github.com/nyanp/tiny-cnn/pull/181